src/plugins/analysis/ipc/test/test_ipc_analyzer.py

```diff
@@ -0,0 +1,22 @@
+from test.unit.analysis.analysis_plugin_test_class import AnalysisPluginTest
+from objects.file import FileObject
+from ..code.ipc_analyzer import AnalysisPlugin
+from pathlib import Path
+
+TEST_DIR = Path(__file__).parent / 'data'
+ 
+class TestAnalysisPluginIpcAnalyzer(AnalysisPluginTest):
+ 
+    PLUGIN_NAME = 'ipc_analyzer'
+    PLUGIN_CLASS = AnalysisPlugin
+ 
+    def test_ipc_system(self):
```

Comment:
this contains two very similar cases. Maybe you can extract the common parts to a function, e.g.
```python
    def _do_ipc_analysis(self, test_file, expected_result):
        test_object = FileObject(file_path=str((TEST_DIR / test_file).resolve()))
        result = self.analysis_plugin.process_object(test_object)
        assert result.processed_analysis[self.PLUGIN_NAME]['full']['ipcCalls'] == expected_result
```

Reply:
Thanks I have extracted the common parts into this function.